spec/swagger_v2/api_swagger_v2_format-content_type_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+describe 'format, content_type' do
+  include_context "the api entities"
+
+  before :all do
+    module TheApi
```

Comment:
This is fine, however these classes will be declared globally, you might want to change it to a dynamic class, like `Class.new(Grape::API)` or namespace them with something closer to the spec name, like `module ContentTypeSpec ...`.